genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/v4/converters/JobServiceProtoConverter.java

```diff
@@ -183,7 +183,8 @@ public JobSpecification toJobSpecificationDTO(
                 .collect(Collectors.toList()),
             protoSpec.getEnvironmentVariablesMap(),
             protoSpec.getIsInteractive(),
-            new File(protoSpec.getJobDirectoryLocation())
+            new File(protoSpec.getJobDirectoryLocation()),
+            protoSpec.getArchiveLocation()
```

Comment:
There's no null in proto fields of type string. This is not going to work.

Reply:
I think you are reading this the other way. This one is converting from protoSpec.

Reply:
Comment applies to conversion both ways.